test/functionalTest/cases/1048_subscription_cache/refresh_cache_wins_notif_disabled_with_update_in_the_middle.test

```diff
@@ -0,0 +1,444 @@
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Cache refresh keeps newer information from cache in disabled notification scenario with sub update in the middle
+
+--SHELL-INIT--
+dbInit CB
+
+mongoCmd ${CB_DB_NAME} 'db.csubs.insert({
+        "_id" : ObjectId("61960a51596aa2623032357d"),
+        "expiration" : NumberLong("9223372036854775807"),
+        "reference" : "http://localhost:9997/notify",
+        "custom" : false,
+        "timeout" : NumberLong(0),
+        "throttling" : NumberLong(0),
+        "servicePath" : "/",
+        "status" : "active",
+        "statusLastChange" : 1546337410,
+        "entities" : [
+                {
+                        "id" : "E",
+                        "isPattern" : "false",
+                        "type" : "T",
+                        "isTypePattern" : false
+                }
+        ],
+        "attrs" : [ ],
+        "metadata" : [ ],
+        "blacklist" : false,
+        "onlyChanged" : false,
+        "conditions" : [ ],
+        "expression" : {
+                "q" : "",
+                "mq" : "",
+                "geometry" : "",
+                "coords" : "",
+                "georel" : ""
+        },
+        "format" : "normalized",
+        "maxFailsLimit": NumberLong(1)
+})'
+
+# t 100 -> MongoDB interactions
+# t 212 -> Cache sync each 5 seconds
+brokerStart CB 100,212 IPv4 -subCacheIval 5 --cache
+
+--SHELL--
+
+#
+# FIXME: regex expression for "the present" dates use 202* pattern. Thus, this
+# test will be next a fix in January 1st, 2030 for the next ten years ;)
+#
+# Relevant fields in step 00
+# - maxFailsLimit 1
+# - status:       active (last changed: 2019-01-01T10:10:10Z)
+#
+# 00. (Done before starting Context Broker) Insert crafted subscription in DB with maxFailsLimit 1 and status active changed in the past
+# 01. GET /v2/subscriptions and see maxFailsLimit 1, status active
+# 02. Upsert entity E twice to trigger notifications to overpass maxFailsLimit
+# 03. Upsert entity E to trigger notification which would be blocked as status in cache is inactive
```

Comment:
Fixed in b4ee08b

Reply:
Actually, it is checked in step 04. If notification weren't blocked at step 03, then in step 04 we will see timesSent: 3, failsCounter: 3 instead of timesSent: 2, failsCounter: 2

Reply:
I mean using the accumulator and checking the notification is sent or not as HTTP request.

Reply:
In my opinion, that would be over-complicating the test. We are plenty of test checking notifications payloads... in this one the focus in the dynamics of the cache so I think it is not necessary.

Reply:
Ok, NTC